docker/Dockerfile.auto

```diff
@@ -0,0 +1,30 @@
+FROM pytorch/pytorch:2.1.0-cuda11.8-cudnn8-runtime
+
+# Set the working directory in the container
+WORKDIR /usr/src/app
+
+# Install build tools and compilers
```

Comment:
remove build tools - you only need them for llama cpp

Reply:
RUN apt-get update

Reply:
And on that note also add llama cpp dockerfile please :)

Reply:
fixed

Reply:
added